chemicaltoolbox/rdock/rbdock.xml

```diff
@@ -7,7 +7,15 @@
     <command><![CDATA[
 ln -s '$active_site' receptor.as &&
 ln -s '$receptor' receptor.mol2 &&
-ln -s $receptor_prm receptor.prm &&
+
+#if $custom_rec.custom_rec_select == 'true':
+    ## simple check that the receptor file name is correct
+    grep -q 'RECEPTOR_FILE receptor.mol2' $custom_rec.prm_file &&
```

Comment:
only thing that could be improved is to provide a proper error message, not sure how to do this in a simple way. But the user is warned clearly in the help for the file input

Reply:
should we fail or straight replace?

Reply:
I guess something like that could work `grep "regex" "filepath" || echo "no match" >&2`

Reply:
I prefer fail so the user is informed something was wrong

Reply:
fair enough!